lib/ruby_lsp/standard/wraps_built_in_lsp_standardizer.rb

```diff
@@ -20,6 +20,13 @@ def run_diagnostic(uri, document)
         @standardizer.offenses(uri_to_path(uri), document.source, document.encoding)
       end
 
+      def run_range_formatting(_uri, _partial_source, _base_indentation)
+        # Not yet supported. Should return the formatted version of `partial_source` is a partial selection of the
```

Comment:
Missing a word here?
```suggestion
        # Not yet supported. Should return the formatted version of `partial_source` which is a partial selection of the
```